static/app/bootstrap/renderDom.tsx

```diff
@@ -13,5 +13,6 @@ export function renderDom(
     return;
   }
 
-  render(<Component {...props} />, rootEl);
+  const root = createRoot(rootEl);
```

Comment:
Does this mean we opt into the new concurrent renderer as well?

Reply:
currently that's the plan

Reply:
okay the new plan is to put it behind an option